kokoro/macos_external/kokoro_build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+# Display commands being run.
+set -x
+
+export LC_CTYPE=en_US.UTF-8
+export TRAVIS_PULL_REQUEST=""
+
+# Code under repo is checked out to ${KOKORO_ARTIFACTS_DIR}/github.
+# The final directory name in this path is determined by the scm name specified
+# in the job configuration.
+cd ${KOKORO_ARTIFACTS_DIR}/github/firebase-ios-sdk
+bundle install
```

Comment:
This ignores essentially everything we've learned while getting Travis working. I don't think we should rewrite all the build scripts.

Builds in travis have two parts: [`install_prereqs.sh`](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/install_prereqs.sh) and [`build.sh`](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/build.sh). There's considerable shared code between projects that I don't think we should be repeating per product.

Reply:
Would it be possible to move this script and associated cfg into the location that would actually be expected for a Core build of some kind? As it stands I can't really visualize how your instructions would actually work for any given build.

For example:
  * The readme mentions a `pull_request` directory but this doesn't use it
  * There's no equivalent directory requirement for continuous builds; is that intended?
  * Is the shell script name supposed to match the .cfg name? (If so kokoro_build.sh matches nothing)
  * Or maybe we're supposed to pass product/platform/method via build_params?

I'm having trouble seeing how this fits together.